### PR TITLE
Fix buttons on new album modal window

### PR DESCRIPTION
**Pull request by JuliaKirschenheuter**
* Part of https://github.com/nextcloud/server/issues/36965
* New album modal window

## Summary

Fix fit of buttons on new album modal window

🏚️ Before | 🏡 After
---|---
image | Screenshot from 2023-10-17 15-54-36

### CODE REVIEW

File: src/components/Albums/AlbumForm.vue [hunk 1]
```diff
@@ -292,9 +292,6 @@ export default {
 	}
 
 	.form-buttons {
-		display: flex;
-		justify-content: space-between;
-
```
Comment (artonge) [inline src/components/Albums/AlbumForm.vue:297]: Or `flex-direction: column` ?

**Review by artonge — APPROVED** [review]

File: src/components/Albums/AlbumForm.vue [hunk 2]
```diff
@@ -294,6 +294,7 @@ export default {
 	.form-buttons {
 		display: flex;
 		justify-content: space-between;
+		flex-direction: column;
```
Comment (szaimen) [inline src/components/Albums/AlbumForm.vue:297]: always? I thought only on mobile?

Comment (JuliaKirschenheuter) [reply src/components/Albums/AlbumForm.vue:297]: Screenshot from 2023-10-13 11-18-04

desktop

Comment (szaimen) [reply src/components/Albums/AlbumForm.vue:297]: Or mybe just use flex-wrap? Wdyt?

Comment (szaimen) [reply src/components/Albums/AlbumForm.vue:297]:
```suggestion
		flex-wrap: wrap;
```

Comment (JuliaKirschenheuter) [reply src/components/Albums/AlbumForm.vue:297]: @szaimen nope ;(
Screenshot from 2023-10-16 15-56-37

Comment (szaimen) [reply src/components/Albums/AlbumForm.vue:297]: Hm... Is there some reason why that does not work? Should usually be shown in the inspector tools...

Comment (JuliaKirschenheuter) [reply src/components/Albums/AlbumForm.vue:297]: does it works for you? For me: styles are there in dev tools.

File: src/components/Albums/AlbumForm.vue [hunk 3]
```diff
@@ -312,4 +313,12 @@ export default {
 .left-buttons {
 	flex-grow: 1;
 }
+
+@media (max-width: 1280px) {
```
Comment (ShGKme) [inline src/components/Albums/AlbumForm.vue:317]: To be a bit more consistent with other places in Photos and 1024 breakpoint.

```suggestion
@media only screen and (max-width: 1020px)
```

Comment (szaimen) [reply src/components/Albums/AlbumForm.vue:317]:
```suggestion
@media (max-width: 1024px) {
```

**Pull request merged**